### ISSUE TITLE
Optional fields 2 in JabRef 3.1 are not configurable

### ISSUE THREAD

**Issue by etilli33** [JabRef/jabref#630]
Running JabRef 3.1 on MacOs X 10.11 in Biblatex mode

The runner Optional fields 2 is empty. When trying to configure e.g. entry type Book, I can delete field names from either Required fields or Optional fields, but all changes done to Optional fields 2 are lost immediately after clicking Apply/OK. I would like to populate it with some own fields, because Optional fields is so full of different fields that it demands scrolling.

I would remember that I was able to change fields in Optional fields 2 back in version 2.7 (?), but have not been able to do this afterwards.

I also thought to reset the settings of e.g. Book entry type but after a restart the Optional 2 tab is again empty.

**Comment by etilli33** [comment]
OK, the issue seems now fixed: I exported the preferences, switched manually Biblatex mode off (there seems to missing a switch in the preference pane?), imported the preferences. Then again switched Biblatex mode on in the exported preferences, imported them. Now after resetting the book entry to default, fields are shown also in Optional field 2 tab.
It seems this issue was only valid for entry types, whose fields I had manually changed, because entry types like InProceeding which I had not changed, had the working Optional fields 2 all the time.
